ci/etc/expected_install_directories

```diff
@@ -56,6 +56,10 @@
 ./include/google/cloud/eventarc/publishing
 ./include/google/cloud/eventarc/publishing/v1
 ./include/google/cloud/eventarc/v1
+./include/google/cloud/gkehub
```

Comment:
Seems like you are missing a few directories (not in this file, in `ci/.../cmake-install.sh`)